sources/models/shared-defaults/oci-defaults-docker.toml

```
@@ -0,0 +1,10 @@
+[metadata.settings.oci-defaults]
+affected-services = ["oci-defaults", "docker"]
+
+[services.oci-defaults]
+configuration-files = ["oci-defaults"]
+restart-commands = []
+
+[configuration-files.oci-defaults]
+path = "/etc/docker/daemon.json"
+template-path = "/usr/share/templates/docker-daemon-json"
```

The docker service already owns this template; we just need to restart it. Otherwise we'd render the same template twice.

```suggestion
[metadata.settings.oci-defaults]
affected-services = ["docker"]
```